data/schema.rdfa

```diff
@@ -5376,7 +5376,7 @@ Unregistered or niche encoding and file formats can be indicated instead via the
 
     <div typeof="rdf:Property" resource="http://schema.org/isBasedOn">
       <span class="h" property="rdfs:label">isBasedOn</span>
-      <span property="rdfs:comment">A resource that was used in the creation of this resource. This term can be repeated for multiple sources. For example, http://example.com/great-multiplication-intro.html.</span>
+      <span property="rdfs:comment">A resource from which this work is derived or of which it is a modification or adaption.</span>
```

Comment:
"from which"..."from which" would be preferred, instead of "from which"..."of which"